### DISCUSSION TITLE
Website monitoring Is it possible?

### DISCUSSION THREAD

**Discussion by orelvis15** [openstatusHQ/openstatus#955] [General]
Is it possible to monitor web pages? I have a use case in which I need to monitor several websites and if they go down, notify an email list.

**Comment by thibaultleouay** [comment]
Hello @orelvis15 

Sure website monitoring is possible, you can monitor anything API and/or Website 

And if it goes down it will trigger an alert and send an email

**Comment by orelvis15** [comment]
HI @thibaultleouay 

I tried to create a monitor for websites but it tells me that the url is not correct, do you have any documentation on how to create monitors for websites?

**Comment by orelvis15** [comment]
The truth is that it is an internal website, I just copied the link of the site on the monitor but it didn't work, sorry, at the moment I don't have access to a screenshot.

I tried **uptime-kuma** also locally on my PC and it worked with the site, with the same url.

**Reply by thibaultleouay** [reply]
@orelvis15 
We are using external probes to check your monitor, your website need to be access from the outside. 

We also provide global monitoring which you can not do with uptime kuma, are probes is deployed in 35 regions :) 

Can you share the link of the url you are trying to monitor ?
